### ISSUE TITLE
Incorrect status reported

### ISSUE THREAD

**Issue by mackensen** [wp-cli/wp-super-cache-cli#11]
I'm attempting to use wp-super-cache-cli in a WordPress 4.5.1 subdomain multisite environment with PHP 5.5. WP Super Cache is installed, configured, and preloading. No matter what I do with the plugin, it reports the following: `Error: The WP Super Cache is not enabled.`. My limited troubleshooting so far suggests that the values from `wp-cache-config.php` aren't available to it. I haven't gotten further than that.

**Comment by halburgiss** [comment]
Similar issue with 4.5.3, and updated plugins. Plugin reports "enabled", but any action fails with "not enabled". wp-cache-config.php is not being created / generated. File permissions are loose enough to allow that.  WP_CACHE is set to TRUE (by me) in wp-config.php. No errors in terminal, and nothing in the Apache / php error logs.

**Comment by iandunn** [comment]
I'm seeing similar errors. I took a quick look at the package code, but it's not immediately obvious why it's failing.

* `status` - `get_option( 'supercache_stats' );` is `false`
* `enable`/`disable` - `global $super_cache_enabled` is `NULL` when it enters the function every time.

It doesn't look like there've been any recent changes to WP Super Cache that would break this, but I didn't look too closely.

I've used this successfully in single-site environments in the past, so it sounds like maybe it's just not compatible with Multisite. I haven't looked close enough to see how WP Super Cache stores config etc for Multisite, though.
